trulens_eval/trulens_eval/feedback/prompts.py

```diff
@@ -121,7 +121,7 @@
 
 STEREOTYPES_PROMPT = v2.Stereotypes.prompt.template
 
-SUMMARIZATION_PROMPT = """
```

Comment:
just curious - what's the reason for renaming summarization prompt to comprehensiveness?

Reply:
Talking to users and customers, comprehensiveness is the standard way that this eval is commonly described. It's also more descriptive than "summarization" on what it does.